tasks.py

```diff
@@ -8,95 +8,96 @@
 
 
 @task(optional=["args"], help={"args": "pytest additional arguments"})
-def test(ctx: Context, args: str | None = ""):
+def test(ctx: Context, args: str = ""):
     """run tests (without coverage)"""
     ctx.run(f"pytest {args}", pty=use_pty)
 
 
 @task(optional=["args"], help={"args": "pytest additional arguments"})
-def test_cov(ctx: Context, args: str | None = ""):
+def test_cov(ctx: Context, args: str = ""):
     """run test vith coverage"""
     ctx.run(f"coverage run -m pytest {args}", pty=use_pty)
 
 
-@task()
-def report_cov(ctx: Context):
+@task(optional=["html"], help={"html": "flag to export html report"})
+def report_cov(ctx: Context, *, html: bool = False):
     """report coverage"""
     ctx.run("coverage combine", warn=True, pty=use_pty)
     ctx.run("coverage report --show-missing", pty=use_pty)
+    if html:
+        ctx.run("coverage html", pty=use_pty)
 
 
-@task(optional=["args"], help={"args": "pytest additional arguments"})
-def coverage(ctx: Context, args: str | None = ""):
+@task(
+    optional=["args", "html"],
+    help={
+        "args": "pytest additional arguments",
+        "html": "flag to export html report",
+    },
+)
+def coverage(ctx: Context, args: str = "", *, html: bool = False):
```

Comment:
With this the html option is not exposed to the hatch commands. I suggest we add an `html` one like

```toml
[tool.hatch.envs.test.scripts]
...
report-cov = "inv report-cov --html '{args}'"
html = "inv coverage --html --args '{args}'"
```

would be used as `hatch run test:html` which seems quite convenient